File: kyu8/you-only-need-one-beginner.php
<?php 
/*
DESCRIPTION:
You will be given an array a and a value x. All you need to do is check whether the provided array contains the value.

Array can contain numbers or strings. X can be either.

Return true if the array contains the value, false if not.
*/
function solution($a, $x) {
    for ($i = 0; $i< count($a); $i++){
      switch($a[$i]){
        case $x:
        return true;
        }
    }
    return false;
  }

File: kyu8/you-only-need-one-beginner.swift
/*
You will be given an array a and a value x. All you need to do is check whether the provided array contains the value.

The type of a and x can be String or Int.

Return true if the array contains the value, false if not.
*/
func check<T: Equatable>(_ a: [T], _ x: T) -> Bool {
    // Your code here
  return a.contains(x)
}
